portal/import_template_ui.php

```diff
@@ -431,18 +464,69 @@ function checkCategory() {
                         }
                     }
                     if (empty($template_id)) {
-                        echo '<tr><td>' . $none_message . "</td></tr>\n";
+                        echo '<tr><td></td><td>' . $none_message . "</td></tr>\n";
+                    }
+                    echo "</tbody>\n";
+                    echo "</table>\n";
+                    ?>
+                </div>
+            </div>
+            <hr />
+            <!-- Send Profiles -->
+            <div class='row'>
+                <div class='col col-12'>
+                    <div class="h5">
+                        <i class='fa fa-eye mr-1' data-toggle='collapse' data-target='#profile_send_collapse' role='button' title="<?php echo xlt('Click to expand or collapse Send Profile panel.'); ?>"></i><?php echo xlt('Send Profiles') ?>
```

Comment:
xla for the title instead of xlt
(guessing that this is going to be the only escaping issue i will be able to point out during this code review :) )